components/question/Outro/index.jsx

```diff
@@ -0,0 +1,42 @@
+import { useRouter } from 'next/router';
+import { useEffect } from 'react';
+
+import Image from '@/components/Image';
+import circleCheckGreenAnimation from '@/public/circle-check-green-animation.gif';
```

Comment:
nice work but just curious, how did you export this Gif from Figma? I only see the option to export SVG 😅

Reply:
downloaded the file 😄 

![image](https://user-images.githubusercontent.com/39977805/169211074-8678dd16-fd3a-4d93-b703-4636b95f3947.png)